health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -201,72 +201,114 @@ const CreateChecklist = () => {
-    const generateCode = (question, prefix, index) => {
-      // Determine the code based on the index
+    const generateCode = (question, prefix, index, parentCounter = '') => {
+      // Generate code regardless of isActive status
       let code = '';
       if (question.parentId === null) {
-        code = `SN${index + 1}`; // Top-level questions
+        code = `SN${index + 1}`;
+        // Only increment counter for active questions
+        if (question.isActive) {
+          activeCounters.top += 1;
+          parentCounter = String(activeCounters.top);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)